app/uk/gov/hmrc/brm/utils/BrmException.scala

```diff
@@ -37,44 +37,64 @@ trait BRMException extends Controller {
     }
   }
 
-  def notFoundPF(method: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
+  def notFoundPF(method: String)(implicit payload: Payload): PartialFunction[Throwable, Result] = {
     case Upstream4xxResponse(body, NOT_FOUND, _, _) =>
       logException(method, body, NOT_FOUND)
       Ok(Json.toJson(BirthResponseBuilder.withNoMatch()))
   }
 
-  def forbiddenPF(method: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
+  def forbiddenPF(method: String)(implicit payload: Payload): PartialFunction[Throwable, Result] = {
     case Upstream4xxResponse(body, FORBIDDEN, _, _) =>
       logException(method, body, FORBIDDEN)
       Ok(Json.toJson(BirthResponseBuilder.withNoMatch()))
   }
 
-  def badRequestPF(method: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
+  def badRequestPF(method: String)(implicit payload: Payload): PartialFunction[Throwable, Result] = {
     case Upstream4xxResponse(body, BAD_REQUEST, _, _) =>
       logException(method, body, BAD_REQUEST)
       BadRequest
   }
 
-  def badGatewayPF(method: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
+ def badGatewayPF(method: String)(implicit payload: Payload): PartialFunction[Throwable, Result] = {
     case Upstream5xxResponse(body, BAD_GATEWAY, _) =>
       logException(method, body, BAD_GATEWAY)
```

Comment:
@manishsw for GRO this would return a BadGateway to consuming service? Do we need to change this?